### PR TITLE
OB-1107 Few improvements for comments release

### PR DESCRIPTION
**Pull request by alexandrelissowski**

### CODE REVIEW

**Review by PierrickMartos — APPROVED** [review]

File: content/md/supplier-onboarder/first-steps/supplier-synchronization.md [hunk 1]
```diff
@@ -28,16 +28,13 @@ As soon as all values of one product are complete for one locale, it is automati
...
 ## Comments
 In every product page, you can see two types of products when the PIM user process a proposal.
 * Product comments: visible on the product header,
 * Attribute comments: visible under each attributes.
 
 ::: info
-Green comments are for the validated proposals and red comments are for the rejected proposals.
-If the comment has been created before the release of the statuses, it appears in grey.
---> 
+Blue comments are for the approved proposals and purple comments are for the rejected proposals.
```
Comment (meganelepalud-akeneo) [inline content/md/supplier-onboarder/first-steps/supplier-synchronization.md:37]:
```suggestion
Blue comments are for the approved proposals and purple comments are for the rejected proposals.
:::
```

File: content/md/supplier-onboarder/manage-products/update-products.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ The header panel shows the following product information:
 * A breadcrumb to navigate within the [product model and variants](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html)
 * The number of missing attributes
 * Some filters to directly access attribute groups or missing attributes
+* Comment on the product if there is
```
Comment (bgouraud) [inline content/md/supplier-onboarder/manage-products/update-products.md:31]: It looks like the sentence is not finished here?

File: content/md/supplier-onboarder/first-steps/supplier-synchronization.md (hunk 1 above)
Comment (meganelepalud-akeneo) [reply content/md/supplier-onboarder/first-steps/supplier-synchronization.md:37]: It seems you forgot to close the info pannel. To be checked in local maybe?

**Review by bgouraud — CHANGES_REQUESTED** [review]

File: content/md/supplier-onboarder/manage-products/update-products.md (hunk 2 above)
Comment (meganelepalud-akeneo) [reply content/md/supplier-onboarder/manage-products/update-products.md:31]: Same reaction

File: content/md/supplier-onboarder/first-steps/supplier-synchronization.md (hunk 1 above)
Comment (alexandrelissowski) [reply content/md/supplier-onboarder/first-steps/supplier-synchronization.md:37]: you're right

File: content/md/supplier-onboarder/manage-products/update-products.md (hunk 2 above)
Comment (alexandrelissowski) [reply content/md/supplier-onboarder/manage-products/update-products.md:31]: "Comment on the product if there is one" would be better

Comment (alexandrelissowski) [reply content/md/supplier-onboarder/manage-products/update-products.md]: changed, tell me if it's OK

**Review by bgouraud — APPROVED** [review]
LGTM

**Pull request merged**